src/AIconLabel.cpp

```diff
@@ -10,7 +10,14 @@ AIconLabel::AIconLabel(const Json::Value &config, const std::string &name, const
     : ALabel(config, name, id, format, interval, ellipsize, enable_click, enable_scroll) {
   event_box_.remove();
   box_.set_orientation(Gtk::Orientation::ORIENTATION_HORIZONTAL);
-  box_.set_spacing(8);
+
+  // set aesthetic default spacing
+  int spacing = config_["icon-spacing"].isInt() ? config_["icon-spacing"].asInt() : -5;
+  box_.set_spacing(spacing);
+
+  int margin_top = config_["margin-top"].isInt() ? config_["margin-top"].asInt() : 6;
+  box_.set_margin_top(margin_top);
```

Comment:
Doesn't that push down the text as well?

Reply:
Yes, this is the goal, since currently, there is no means to push down both the icon and text - just the text. Adding this configuration option is supposed to circumvent this. This also allows the user to be flexible with the position of the text relative to the image, as even with this, I want the text to be 1px higher than the default which I can change in `styles.css`